### ISSUE TITLE
ng2-auto-complete drop down does not updating search list on changing source array

### ISSUE THREAD

**Issue by pravinrahurkar-tudip** [ng2-ui/auto-complete#302]
Hi,

I am facing issue with updating source data of ng2-auto-complete drop down. It shows the previous source in the drop list.

**Here are the steps to reproduce the issue:** 

- First, add source data to dropdown and focus on the drop down it shows all the data properly.
- Then on typing 3 or more than 3 characters to filter data, I have fetched the data from API service.
- And appended it to source data but it is not reflecting the latest data in search dropdown.
- Also, when I focus out from the ng2-auto-complete field and again focus on it, the dropdown list shows the latest fetched data.

**Expected Behaviour:**

- After changing the source data to fetched data from API, without focus out and focus in ng2-auto-complete dropdown should display latest fetched data.
- Basically, the DOM is not updating to latest fetched data.

Any help is very much appreciated here.

Regards,
Pravin Rahurkar.

**Comment by pitiriakova** [comment]
Hello, I had the same issue. It was helpful for me
http://plnkr.co/edit/ExzNSh?p=preview
